mappings/net/minecraft/world/biome/layer/samplingUtil/CachingLayerSampler.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cgh net/minecraft/world/biome/layer/CachingLayerSampler
+CLASS cgh net/minecraft/world/biome/layer/samplingUtil/CachingLayerSampler
```

Comment:
Same here

Reply:
Packages should be lowercase according to Java conventions. https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html